src/trendlines/utils.py

```diff
@@ -67,25 +67,33 @@ def get_metric_parent(metric):
 
 def format_metric_for_jstree(metric):
     """
-    Format a metric name into a dict consumable by jsTree.
+    Format a metric into a dict consumable by jsTree.
 
     See "Alternative JSON format" in the `jsTree docs`_.
 
     .. _`jsTree docs`: https://www.jstree.com/docs/json/
 
     Parameters
     ----------
-    metric : str
-        The metric name to format.
+    metric : :class:`orm.Metric`
+        The metric to format.
 
     Returns
     -------
     dict
-        A dict with the following keys: id, parent, text, url
+        A dict with the following keys: id, parent, text, url, metric_id
     """
+    metric_id = metric.metric_id
+    metric = metric.name
     parent = get_metric_parent(metric)
     url = url_for('pages.plot', metric=metric)
-    return {"id": metric, "parent": parent, "text": metric, "url": url}
+    jstree_node = {
+        "id": metric,
+        "parent": parent,
+        "text": metric,
+        "metric_id": metric_id,
+    }
+    return jstree_node
```

Comment:
I overwrote `metric` at the start of this function to reduce the size of the diff, but I think it would be more readable if I instead changed variable names. Plus we don't need `url = url_for(...` anymore.

```python
metric_name = metric.name
parent = get_metric_parent(metric_name)
jstree_node = {
    "id": metric_name,
    "parent": parent,
    "text": metric_name,
    "metric_id": metric.metric_id,
}
return jstree_node